docs/insomnia/introduction-to-plugins.md

```diff
@@ -91,24 +106,85 @@ Example: Plugin package.json file
 }
 ```
 
+### Alter a Response Body
+
+The response body works with [NodeJS Buffers](https://nodejs.org/api/buffer.html). When developing a plugin, you may want to write an updated API and work with Buffers to change the request body. 
```

Comment:
```suggestion
The response body works with [NodeJS Buffers](https://nodejs.org/api/buffer.html). In order to change the response body via a plugin, you will need to translate to/from a Buffer.
```

Just a suggestion here, pls reword as appropriate!

Reply:
Yes, I think we should develop on the Plugin docs. For now, I have slightly altered the language.